### PR TITLE
BaSP-TG-1: Update backend schemas

### PR DESCRIPTION
**Pull request by mara666**
- Install cors dependency
- Add "status" property to Employee, Admin, Project Schemas
- Update Status at Postman Collections -(Employee, Admin, Project)
- Add "status" property inactive (false value) to controllers POST methods
- Update Models with "status" property
- Add rate, role and employee id to project schemas
- Update project seeders with rate, role and employee id
- Update Project Postman Collection (rate-role-employee Id)
- Update Project Collection and populate employees info
- Update models with employees info at Projects
- Update Joi validations for Employee, Admin, Project
- Fix admins test

### CODE REVIEW

**Review by FabianDutraSilva — COMMENTED** [review]
Checked, correctly

**Review by FabianDutraSilva — COMMENTED** [review]
Mongo db degraded

File: src/validations/admins.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ const validateCreation = (req, res, next) => {
     email: Joi.string()
       .email({ minDomainSegments: 2, tlds: { allow: ['com', 'net'] } }),
     password: Joi.string().min(10).max(30).required(true),
+    status: Joi.boolean(),
```
Comment (Maxi-SR) [inline src/validations/admins.js:10]: The status should be required here, right?

File: src/validations/admins.js [hunk 2]
```diff
@@ -27,6 +28,7 @@ const validateEdition = (req, res, next) => {
     email: Joi.string()
       .email({ minDomainSegments: 2, tlds: { allow: ['com', 'net'] } }),
     password: Joi.string().min(10).max(30),
+    status: Joi.boolean().required(),
```
Comment (Juanoauil) [inline src/validations/admins.js:31]: its required, is correct?

File: src/validations/admins.js (hunk 1 above)
Comment (Juanoauil) [reply src/validations/admins.js:10]: It shouldn't be required because we don't send it from FE

File: src/models/Projects.js [hunk 3]
```diff
@@ -9,7 +9,13 @@ const projectSchema = new Schema(
     endDate: { type: Date, required: true },
     description: { type: String, required: true },
     clientName: { type: String, required: true },
-    employees: [{ type: Schema.Types.ObjectId, ref: 'Employee' }],
+    employees: [{
+      _id: false,
```
Comment (Juanoauil) [inline src/models/Projects.js:13]: is it correct?

**Review by Juanoauil — APPROVED** [review]

**Review by DamianPalavecino — APPROVED** [review]
Approved, everything looks ok

**Pull request merged**